index.html

```diff
@@ -210,27 +213,33 @@ <h3>Languages</h3>
 </ul>
 
     <ul class="lang-skills">
-        <li> <p>Frameworks</p> <img src="Images/right-arrow.png" alt=""></li>
-        <li class="skills"> <p>Skills</p> <img src="Images/right-arrow.png" alt=""></li>
+        <li class="framework"> 
+            Frameworks <img class="image" src="Images/right-arrow.png" alt="">
+        </li> 
+        <li class="skills">
+            Skills <img class="image" src="Images/right-arrow.png" alt="">
+        </li>
     </ul>
     <div><img class="shape" src="Images/Separator Bottom.png" alt=""></div>
      <div><img class="shape-1" src="Images/Separator Bottom.png" alt=""></div> 
 
 </section>
-<section class="contact">
+<!-- This is the contact page -->
+<section id="contact">
     <h2>Contact Me</h2> 
 <p>
     If you have an application you are interested in developing, 
     a feature that you need built or a project that needs coding.
      I’d love to help with it 
 </p>
-   <input placeholder="Yeremias" type="text">
-   <input placeholder="natajayanj14@gmail.com" type="text">
-   <textarea placeholder="Write your message here" name="" id="" cols="30" rows="10"></textarea>
-
-<button  class="contact-button"><a href="">Get in touch</a></button>
-
-<div class=""><img src="Images/Shape.png" alt=""></div>
+<form action="https://formspree.io/f/xgebkzlz" method="post">
+   <input name="Name" placeholder="Yeremias" type="text" maxlength="30" required>
+   <input name="Email" placeholder="natajayanj14@gmail.com" type="email" required>
+   <textarea placeholder="Write your message here" name="Message" id="" cols="30" rows="10" maxlength="500" type="text" required></textarea>
+<button type="submit" id="contact-button"><a href="">Get in touch</a></button>
```

Comment:
- [ ] I have noticed that your form is not able to submit 😞 . I have noticed that you have used the `a` tag inside the `button` tag. This is not Semantically correct and therefore the form won't be submitted. Kindly consider removing it and verify the form submission ☺️